index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+	<meta charset="UTF-8">
+	<title>Womarkers Code - Adail Sena</title>
+	
+</head>
+<body>
+	
+		<div id="menu">
+		<nav>
+		<p style="background-color:black;"</p>
+		
+			<img src="images/logo.png" alt="Logotipo do Womarkers Code" width:"200px" height="66px" >
+					
+            <a href="index.html" font color="#ffffff" text-decoration: none>HOME</font></a>
```

Comment:
O site é de página única, por isso você tem que colocar a referência para as seções, não para outros arquivos.
```suggestion
            <a href="#banner" font color="#ffffff" text-decoration: none>HOME</font></a>
```